src/Components/ToDoItem/ToDoItem.tsx

```diff
@@ -1,8 +1,15 @@
 import React from 'react';
+import { default as toDoItemStyles} from './ToDoItem.module.scss';
+import bemCssModule from 'bem-css-modules';
 
-const ToDoItem = () => {
+const style = bemCssModule(toDoItemStyles);
+

```

Comment:
You should add Props interface that will include all properties used by TodoItem component.